recipes/trx-python/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "trx-python" %}
+{% set version = "0.2.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/trx-python-{{ version }}.tar.gz
+  sha256: 16b4104d7c991879c601f60e8d587decac50ce60388aae8d0c754a92136d1caf
+
+build:
+  skip: true  # [py<38]
```

Comment:
This one has scripts and cannot be noarch. If you get a new version, please remember to check if they dropped the scripts upstream so we can convert it to noarch.